test/metabase/util/malli_test.clj

```diff
@@ -1,63 +0,0 @@
-(ns ^:mb/once metabase.util.malli-test
```

Comment:
Removed and combined into `malli_test.cljc` -- it was messing things up to have this namespace exist twice in a `.clj` file and a `.cljc` file